public/index.html

```diff
@@ -26,7 +26,7 @@
     -->
 
   <title>Cow Swap</title>
-  <script async src="https://w.appzi.io/w.js?token=5ju0G"></script>
+  <!-- <script async src="https://w.appzi.io/w.js?token=5ju0G"></script> -->
```

Comment:
hmmmm, you not removing it only on dev...

Reply:
too fast, i didnt commit last thingy